Docs/Notifications.md

```diff
@@ -0,0 +1,58 @@
+# Menu Notifications
+
+Home notifications are now available from v1.4.0. These are shown to the user on the main menu screen of the game and are refreshed everytime the menu is loaded.
+![image](https://github.com/SideswipeeZ/YgoMaster/assets/20016930/f15c1a98-6bcd-4615-bebc-b4b3adb42645)
```

Comment:
Could you add the image to the Docs/Pics folder and link it like `![image](Pics/IMG_NAME.png)`. That way the image is part of the repo.

The note about v1.4.0 probably isn't needed but if keeping it it'd be better to say v1.40 to avoid confusion with the client versioning which uses the X.X.X format.